events/iwslt-2024.md

```diff
@@ -0,0 +1,77 @@
+---
+autogenerated: true
+autogenerated_from: events.json
+parent: Events
+nav_order: 2024
+layout: event
+title: IWSLT 2024
+endDate: '2024-08-16'
+name: IWSLT 2024
+id: iwslt-2024
+description: International Conference on Spoken Language Translation (IWSLT)
+calls_for_papers_deadline: '2024-04-29'
+start_date: '2024-08-15'
+end_date: '2024-08-16'
+location:
+  name: Bangkok, Thailand
+  type: Place
+  event_attendance_mode: MixedEventAttendanceMode
+organizer:
+  type: Person
+  url: https://iwslt.org/2023/#organizers
+future_tense_opening_paragraph: The International Conference on Spoken Language Translation
+  (<strong>IWSLT</strong>) is an annual scientific conference on spoken language translation.<br>
+  <br> IWSLT 2024 will be co-located at ACL 2024 online and in Bangkok, Thailand from
+  15 to 16 August, 2024. <br><br> IWSLT 2024 will be a hybrid event.
+past_tense_opening_paragraph: The International Conference on Spoken Language Translation
+  (<strong>IWSLT</strong>) is an annual scientific conference on spoken language translation.<br>
+  <br> IWSLT 2024 was co-located at ACL 2024 online and in Bangkok, Thailand from
+  15 to 16 August, 2024. <br><br> IWSLT 2024 was a hybrid event.
+links:
+- <a href='https://iwslt.org/2024/'>iwslt.org/2024/</a>
+- 'This is from **markdown**: (iwslt.org/2024/)[https://iwslt.org/2024/]'
+important_dates:
+  Release of shared task training and developer data: '2024-01-15'
+  Evaluation: 1 - 15 April
+  Paper submission: '2024-04-29'
+  Notification of acceptance: '2024-06-04'
+  Camera ready deadline: '2024-06-24'
+  Pre-recorded video due: '2024-07-22'
+  Conference: 15 - 16 August
+speakers:
+- type: Organizers
+  about:
+  - Marine Carpuat, University of Maryland, United States
+  - Marcello Federico, AWS AI Labs, United States
+  - Alex Waibel, CMU, United States
+  - Jan Niehues, KIT, Germany
+  - "Sebastian St\xFCker, KIT, Germany"
+  - Elizabeth Salesky, JHU, United States
+  - Atul Kr. Ojha, University of Galway, Ireland
+shared_tasks:
+- <a href='https://iwslt.org/2024/s2s'>Speech-to-speech track</a>
```

Comment:
This is hacky.

We should do either like this:

```
shared_takes:
- name: Speech-to-speech track
   url: https://iwslt.org/2024/s2s
...
```

Or like this:
```
shared_takes:
- Speech-to-speech track: https://iwslt.org/2024/s2s
...
```

In JSON the equivalent would be:
```
"shared_tasks": [
      {
        "name": "Speech-to-speech track",
        "url": "https://..."
      }
      ...
  ]
```
or
```
"shared_tasks": {
      "Speech-to-speech track": "https://..."
     ...
}
```